src/Tooltip.js

```diff
@@ -2,38 +2,69 @@ import React from 'react';
 import t from 'prop-types';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 
-import { toolTipStyles, toolTipBodyBtnStyles, toolTipComposeEmailStyles, afterStyles } from './TooltipStyles.js';
+import {
+  toolTipStyles,
+  toolTipInnerWrapper,
+  toolTipHead,
+  toolTipBodyBtnStyles,
+  toolTipBtnText,
+  toolTipAnchorText,
+  toolTipComposeEmailStyles,
+  afterStyles
+} from './TooltipStyles.js';
 
-const openEmailClient = email => {
-  console.log('triggered');
-  document.location = `mailto:${email}`;
-};
-
-const Tooltip = ({ emailAddress, className }) => (
-  <div className={className + '__tooltip'} style={toolTipStyles}>
-    <div className={className + '__tooltip-head'}>{emailAddress}</div>
-    <div className={className + '__tooltip-body'}>
-      <CopyToClipboard text={emailAddress} onCopy={() => alert('copied.')}>
-        <button className={className + '__tooltip--copy-email-address __tooltip-body-btn'} style={toolTipBodyBtnStyles}>
-          Copy Email Address To ClipBoard
-        </button>
-      </CopyToClipboard>
+const Tooltip = props => (
+  <div className={props.className + '__tooltip'} style={toolTipStyles || props.toolTipStyles}>
+    <div
+      className={props.className + '__tooltip-inner-wrapper'}
+      style={toolTipInnerWrapper || props.toolTipInnerWrapper}
+    >
+      <div className={props.className + '__tooltip-head'} style={toolTipHead || props.toolTipHead}>
+        {props.emailAddress}
+      </div>
+      <div className={props.className + '__tooltip-body'}>
+        <CopyToClipboard text={props.emailAddress} onCopy={() => alert('copied.')}>
+          <button
+            className={props.className + '__tooltip--copy-email-address __tooltip-body-btn'}
+            style={toolTipBodyBtnStyles || props.toolTipBtnStyles}
+          >
+            <span
+              className={props.className + '__tooltip-body-btn--text'}
+              style={toolTipBtnText || props.toolTipBtnText}
+            >
+              Copy Email Address To ClipBoard
+            </span>
+          </button>
+        </CopyToClipboard>
 
-      <a
-        href={`mailto:${emailAddress}`}
-        className={className + '__tooltip--compose-default-email __tooltip-compose-default-email'}
-        style={toolTipBodyBtnStyles}
-      >
-        Compose email in default client
-      </a>
+        <a
+          href={`mailto:${props.emailAddress}`}
+          className={props.className + '__tooltip--compose-default-email __tooltip-compose-default-email'}
+          style={toolTipBodyBtnStyles || props.toolTipBodyBtnStyles}
```

Comment:
I think we should merge the two objects? I don't think a user will want to lose all of the default styles if they only over ride one of them. Whatcha think?

Reply:
I think you are right about it. Though I have this idea, like to expose common design changes the user can commonly wants like opening up `fontFamily`, `fontColor` and `backgroundColor`. And in extreme cases letting them change the whole styles in case they wants to.